web_src/css/repo/home.css

```diff
@@ -0,0 +1,37 @@
+.latest-releases {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
```

Comment:
Does `-webkit` make sense?

Reply:
I'm not sure how to limit them into 2 lines.
Do you have some suggestions?

Reply:
Why `height + overflow`  doesn't work?

Reply:
https://github.com/go-gitea/gitea/pull/32213/commits/5dbb11053fdba391be814028ead0ff7806c4a69e